jetcd-core/src/main/java/io/etcd/jetcd/maintenance/StatusResponse.java

```diff
@@ -73,4 +75,40 @@ public long getRaftIndex() {
     public long getRaftTerm() {
         return getResponse().getRaftTerm();
     }
+
+    /**
+     * Returns the current raft applied index of the responding member.
+     *
+     * @return the raft applied index.
+     */
+    public long getRaftAppliedIndex() {
+        return getResponse().getRaftAppliedIndex();
+    }
+
+    /**
+     * Return the information corresponding to the alarms, health and status.
+     *
+     * @return the list of errors
+     */
+    public List<String> getErrorList() {
+        return getResponse().getErrorsList();
+    }
+
+    /**
+     * Returns the size of the backend database logically in use, in bytes, of the responding member.
+     *
+     * @return dbSizeInUse
+     */
+    public long getDbSizeInUse() {
+        return getResponse().getDbSizeInUse();
+    }
+
+    /**
+     * Returns if the member is a raft learner.
+     *
+     * @return if the member is a raft learner
+     */
+    public boolean getIsLearner() {
```

Comment:
Let's rename `getIsLearner()` to `isLearner()` for consistency; both in general but also e.g. specifically with the existing [`Member.isLearner()`](https://github.com/etcd-io/jetcd/blob/9b4c9dd2ce8a78bf9cd99e88b88af139e422ba87/jetcd-core/src/main/java/io/etcd/jetcd/cluster/Member.java#L74)?

```suggestion
    public boolean isLearner() {
```